wip/stacked-borrows.md

```diff
@@ -176,27 +325,52 @@ we determine the extent of memory that this place covers using `size_of_val` and
    Remember the position of the item matching the tag in the stack.
```

Comment:
typo: "we determine the extent of memory" should start with a capital letter: "We determine ..."

Reply:
Thanks! Fixed.